api/helpers/tell-me/contributorParsing.js

```diff
@@ -0,0 +1,168 @@
+/**
```

Comment:
It's a problem when you have bad tools :p

A good IDE is handling well the subpaths and file finding, even with similar filenames
And to avoid the ugly relative imports, we should use absolute ones with babel path aliasing : it's really helpful, more refactorable, less fragile ... :)

Reply:
Any code should be ignorant of the developers tools: no specific IDE-related config should be in a repo, even less an architecture thought related to IDEs. Exactly like OSs, it's better to be agnostic and give the freedom to devs and users. That has always been one of the big weaknesses of the Microsoft / Visual Studio or Java / Eclipse / Jetbrains worlds. You can see where it went haha. I admit I'm more of a Text Editor / vim / emacs because who can do nicely with less can only do better with more ^^.

But to make it short, it's simply better to follow top-tech companies patterns since they are not the best for no reason ;).

As for the path aliases, they're there to make things even easier. I already use them on all my repos obviously (https://github.com/betagouv/tell-me/blob/main/tsconfig.json#L7). But avoiding nested folders still stands.